Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/AppState.java

```diff
@@ -620,6 +636,102 @@ public void changeCalibration(int deviceId, boolean isCalibrated) {
         emitEvent(DataAction.DeviceCalibrationStatus);
     }
 
+    /**
+     * This function will generate the Report string which is to be written in the csv file
+     * @return
+     */
+    public String generateCSVReportString(){

```

Comment:
It looks like these 2 functions are utility functions, not directly related to AppState. Don't you think it's better to move them out of this file?